nested/network.json

```diff
@@ -28,7 +28,7 @@
             }
         },
         {
-            "apiVersion": "2018-02-01",
+            "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.